### PR TITLE
use postgres defaults that work both on Linux and OsX

### PR DESCRIPTION
**Pull request by cguardia**
## Description
Use a database user with a password as default to have same settings in Linux and OsX.

## Motivation and Context
Issue 492.

### CODE REVIEW

File: docs/source/developing_dallinger_setup_guide.rst [hunk 1]
```diff
@@ -74,40 +74,30 @@ Create the Database
...
 ::
 
-    psql -c 'create database dallinger;' -U postgres
+    createuser -P dallinger
+    (Password: dallinger)
+    createdb -O dallinger dallinger
 
-If you get the following error...
+The first command will create a user named ``dallinger`` and prompt you for a
+password. The seconf command will create the ``dallinger`` database, setting
```
Comment (suchow) [inline docs/source/developing_dallinger_setup_guide.rst]: => `second`

**Comment by cguardia** [comment]
This should be ready to go.

**Comment by suchow** [comment]
I think this may have introduced a backwards incompatibility. Specifically, existing users will have a `dallinger` database with an owner `postgres` and no `dallinger` user. To get my system back up and running I ran:

```
createuser dallinger
dropdb dallinger
createdb -O dallinger dallinger
```

Any way around this?

cc: @cguardia @jessesnyder

**Comment by cguardia** [comment]
Opened issue #721 to address this problem.

**Pull request merged**